osnovi_OOP.rb

```diff
@@ -0,0 +1,111 @@
+class Station # класс станция
+  attr_reader :trains, :name # возвращает список поездов, находящися на станции
+
+  def initialize(name_station) # имя станции указывается при создании
+    @name = name_station
+    @trains = [] # массив поездов
+  end
+
+  def add_trains(train) # принимает поезда по одному за раз
```

Comment:
```suggestion
  def add_train(train) # принимает поезда по одному за раз
```
метод принимает один объект